explainer.md

```diff
@@ -463,48 +349,34 @@ For example:
   The screen reader sends the browser an increment *action* on the slider element
   in the accessibility tree.
 
-Currently, browsers implement partial support by accessible actions either by
-implementing built-in support for native HTML elements (for example, a native
-HTML `<input type="range">` already supports increment and decrement actions,
-and text boxes already support actions to set the value or insert text.
+Currently, browsers implement partial support for accessible actions 
+either by implementing built-in support for native HTML elements 
+(for example, a native HTML `<input type="range">` already supports 
+increment and decrement actions,
+and text boxes already support actions to set the value or insert text).
 
 However, there is no way for web authors to listen to accessible actions on
 custom elements.  For example, the custom slider above with a role of `slider`
```

Comment:
I think the mentioned custom slider example is much earlier in the doc. Maybe either include a link back to it or duplicate the snippet?

Reply:
Good catch, done.